monitoring/rpc_stats_interceptor_test.go

```diff
@@ -144,7 +144,7 @@ func TestMultipleOKRequestsTotalLatency(t *testing.T) {
 		}
 	}
 	count, sum := stats.ReqSuccessLatency.Info("testmethod")
-	if wantCount, wantSum := uint64(3), 3837.0; count != wantCount || sum != wantSum {
+	if wantCount, wantSum := uint64(3), 3.8369999999999997; count != wantCount || sum != wantSum {
```

Comment:
Maybe repeat some of the calculation for `wantSum` to avoid the rounding ?  (E.g. something like `time.Duration(3837*time.Millisecond).Seconds()`